ml-agents-envs/mlagents_envs/rpc_utils.py

```diff
@@ -123,13 +123,13 @@ def batched_step_result_from_proto(
         is_visual = len(obs_shape) == 3
         if is_visual:
             obs_shape = cast(Tuple[int, int, int], obs_shape)
-            obs_list += [
+            obs_list.append(
```

Comment:
Found this when looking for other things like what `_get_vis_obs_list()` did. These have the same effect, but don't require creating an intermediate list object, and are clearer (IMHO).